### PR TITLE
French fix

### PR DESCRIPTION
**Pull request by ludoo0d0a**
Here are my last fixes for french, I think

changelogs <1.5 are rollback to EN

### CODE REVIEW

**Review by ludoo0d0a — COMMENTED** [review]
bug: malformed json on windchill_title

File: assets/i18n/fr.json [hunk 1]
```diff
@@ -3132,7 +3132,7 @@
   "whitespace_interpreter": "Interpréteur",
   "whitespace_language_start": "Début",
   "whitespace_language_readable_code": "Code lisible",
-  "windchill_title": "Refroidissement par le vent",
+  "windchill_title": "Refroidissement par le vent
```
Comment (ludoo0d0a) [inline assets/i18n/fr.json]: malformed...

File: assets/i18n/en.json [hunk 2]
```diff
@@ -1630,7 +1630,7 @@
   "icecodes_system_wmo_snow_2": "Up to 10 cm",
   "icecodes_system_wmo_snow_3": "Up to 20 cm",
   "icecodes_system_wmo_snow_4": "Up to 30 cm",
-  "icecodes_system_wmo_snow_5": "Up tou 50 cm",
+  "icecodes_system_wmo_snow_5": "Up to 50 cm",
   "icecodes_system_wmo_snow_6": "Up tou 75 cm",
```
Comment (S-Man42) [inline assets/i18n/en.json:1634]: I guess the other "tou" lines should be changed as well. :)

File: assets/i18n/fr.json (hunk 1 above)
Comment (S-Man42) [reply assets/i18n/fr.json]: Did you change that? It's not clear to me why you removed the ", characters...

**Comment by S-Man42** [comment]
Why the changelog rollback?

File: assets/i18n/fr.json (hunk 1 above)
Comment (ludoo0d0a) [reply assets/i18n/fr.json]: typo, but I cannot fix
I'm searching how to edit a merged commit after PR... grrr

**Comment by ludoo0d0a** [comment]
> Why the changelog rollback?

They were part translated. 
I didn't understand at first, if I should translate all or only 1.5.0
I fix my mistakes here. sorry !

File: assets/i18n/fr.json (hunk 1 above)
Comment (ludoo0d0a) [reply assets/i18n/fr.json]: It should be ok now.

File: assets/i18n/en.json (hunk 2 above)
Comment (ludoo0d0a) [reply assets/i18n/en.json:1634]: nice catch

Comment (ludoo0d0a) [reply assets/i18n/en.json]: done !

**Pull request merged**